Minecraft/Chat.ts

```diff
@@ -0,0 +1,94 @@
+import { DiscordClient } from '../Utils/DiscordClient';
+import { PlayerManager } from './PlayerManager';
+
+export function handleChat(client: DiscordClient) {
+    const { bot } = client;
+
+    // Detects when a player sends a message in minecraft chat
+    bot.on('player_chat', (data) => {
+        let name = JSON.parse(data.networkName).text;
+        name.length
+            ? (name = name)
+            : (name = JSON.parse(data.networkName)?.extra[2]?.text);
+        if (name === bot.username) return; // Ignore messages from the bot
+        if(name.startsWith('N00bBot')) return;
```

Comment:
Consider making a full list of n00bbots as not all usernames starting with `N00bBot` are actual bots (eg. `NoobBot1`)